src/components/command-bar/commands/search/components/tutorials-tab-contents/types.ts

```diff
@@ -11,15 +16,25 @@ interface TutorialsTabContentsProps {
 
 type TutorialHitObject = Hit<{
 	description: string
-	name: string
-}> & {
-	defaultContext: {
-		slug: string
+	page_title: string
+	name: string // TODO: remove this once page_title is fully seeded
+	headings: string[]
+}> &
```

Comment:
One question I have and was curious about the previous implementation. When the types are passed to `Hit` does that reflect the searchable attributes? Or why wasn't `defaultContext` included in that first object?

Reply:
Great question, I'm not sure tbh 🤔 That they might be searchable attributes seems like a good explanation to me 👍 I would think maybe only the `defaultContext.name` or something would be a searchable attribute, so maybe that's why the whole `defaultContext` obj was left out? (i'm speculating, I'm not very familiar with the tutorials index or the command-bar search code here 😅 )